browser/ui/views/side_panel/brave_side_panel_coordinator.cc

```diff
@@ -59,7 +59,7 @@ void BraveSidePanelCoordinator::OnTabStripModelChanged(
   // https://github.com/brave/brave-browser/issues/38331#issuecomment-2119901610
   if (active_tab_changed) {
     auto* old_contextual_registry =
-        SidePanelRegistry::Get(selection.old_contents);
+        SidePanelRegistry::GetDeprecated(selection.old_contents);
```

Comment:
seems like we have a lot of SidePanelRegistry cleanup to do? Does this even make sense or work correctly since we're using the tab scoped side panel?

Reply:
It should work correctly because the upstream code gets the TabInterface from the WebCotnents:
```
// static
SidePanelRegistry* SidePanelRegistry::GetDeprecated(
    content::WebContents* web_contents) {
  tabs::TabInterface* tab = tabs::TabInterface::GetFromContents(web_contents);
  return tab->GetTabFeatures()->side_panel_registry();
}
```
I can create a follow up issue, but note that the upstream change indicates: 
```
      SidePanelRegistry::Get was renamed to
      SidePanelRegistry::GetDeprecated. Not all use cases could be converted
      since many of the callsites do not yet use the Tab primitive.
```

Reply:
> seems like we have a lot of SidePanelRegistry cleanup to do? Does this even make sense or work correctly since we're using the tab scoped side panel?

`[Brave]SidePanelCoordinator` determines which active entry should be shown from global or per-tab registry. Checking content(tab)'s registry here is normal behavior.